backend/src/main/java/com/yagubogu/stadium/service/StadiumService.java

```diff
@@ -0,0 +1,50 @@
+package com.yagubogu.stadium.service;
+
+import com.yagubogu.checkin.repository.CheckInRepository;
+import com.yagubogu.game.domain.Game;
+import com.yagubogu.game.repository.GameRepository;
+import com.yagubogu.global.exception.NotFoundException;
+import com.yagubogu.stat.dto.OccupancyRateTotalResponse;
+import com.yagubogu.stat.dto.OccupancyRateTotalResponse.OccupancyRateResponse;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class StadiumService {
+
+    private final GameRepository gameRepository;
+    private final CheckInRepository checkInRepository;
+
+    public OccupancyRateTotalResponse findOccupancyRate(final long stadiumId, final LocalDate today) {
+        Game game = getGame(stadiumId, today);
+        int checkInPeoples = checkInRepository.countByGame(game);
```

Comment:
C: `peoples`라는 네이밍이 일관적이지 않은 느낌이에요. totalCheckInCounts나 checkInMembers로 수정하는건 어떠신가요?
```suggestion
        int checkInMembers = checkInRepository.countByGame(game);
```